support/android/apk/servoview/src/main/java/org/mozilla/servoview/Servo.java

```diff
@@ -38,10 +39,10 @@ public Servo(
 
         mAssetMgr = activity.getResources().getAssets();
 
-        Callbacks cbs = new Callbacks(client, gfxcb);
+        mServoCallbacks = new Callbacks(client, gfxcb);
```

Comment:
I'm curious about the name change from cbs to mServoCallbacks -- I think it's clearer and more consistent, but (not being familiar with this part of the code), what's up with the 'm' prefix?

Reply:
We moved from a variable to a class field. And it's convention to prefix a class field with a `m`.